frontend/src/components/CheckoutForm.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect } from "react";
+import {
+  CardElement,
+  useStripe,
+  useElements
+} from "@stripe/react-stripe-js";
+import './CardSectionStyle.css'
+import { api } from "../services/api/config";
+import { END_POINTS } from "../services/api/endpoints";
+import { user_config } from "../config/auth";
+import Toast from 'react-bootstrap/Toast'
+import { MdNotificationsActive } from "react-icons/md";
+
+
+const CheckoutForm = (props) => {
+    const [succeeded, setSucceeded] = useState(false);
+    const [error, setError] = useState(null);
+    const [processing, setProcessing] = useState('');
+    const [disabled, setDisabled] = useState(true);
+    const [clientSecret, setClientSecret] = useState('');
+    const [shouldDisplayNotification, setShouldDisplayNotification] = useState(false)
+    const [notification, setNotification] = useState()
+    const stripe = useStripe();
+    const elements = useElements();
+
+
+  const displayNotification = (message, variant) => {
+    setNotification({message, variant})
+    setShouldDisplayNotification(!shouldDisplayNotification)
+  }
+
+    useEffect(() => {
+      (async () => {
+            const {order} = props
+            if(order.totalPrice) {
+                await api.post(END_POINTS.CREATE_PAYMENT_INTENT, {amount: order.totalPrice }, user_config)
+                .then(response => {
+                    setClientSecret(response?.data?.clientSecret)
+                }).catch(err => {
+                    alert(err.message)
+                })
+              }
+            })()
+    }, [ ,props.order]);
+  
+    const cardStyle = {
+      style: {
+        base: {
+          color: "#32325d",
+          fontFamily: 'Arial, sans-serif',
+          fontSmoothing: "antialiased",
+          fontSize: "16px",
+          "::placeholder": {
+            color: "#32325d"
+          }
+        },
+        invalid: {
+          color: "#fa755a",
+          iconColor: "#fa755a"
+        }
+      }
+    };
+  
+    const handleChange = async (event) => {
+      setDisabled(event.empty);
+      setError(event.error ? event.error.message : "");
+    };
+  
+    const handleSubmit = async ev => {
+      ev.preventDefault();
+      setProcessing(true);
+  
+      const payload = await stripe.confirmCardPayment(clientSecret, {
```

Comment:
@giangnguyen13  Here's the code that tells you if the payment was confirmed or not. If there's an error in the payload then it didn't go trough, otherwise it did.